backend/src/packages/auth/auth.controller.ts

```diff
@@ -67,6 +67,13 @@ class AuthController extends Controller {
       body: UserSignUpRequestDto;
     }>,
   ): Promise<ApiHandlerResponse> {
+    if (await this.authService.checkExistingUser(options.body)) {
+      return {
+        status: HttpCode.CONFLICT,
```

Comment:
new property should be declared here `shared/src/libs/packages/http/libs/enums/http-code.enum.ts`. Then run `npm i` to build shared